src/main/java/org/bookkeepervisualmanager/bookkeeper/BookkeeperManager.java

```diff
@@ -149,16 +214,12 @@ public LedgerMetadata getLedgerMetadata(long ledgerId) throws BookkeeperExceptio
     }
 
     public List<Long> getAllLedgers() throws BookkeeperException {
-        try {
-            List<Long> resultLedgers = new ArrayList<>();
-
-            Iterable<Long> ledgersIds = bkAdmin.listLedgers();
-            ledgersIds.forEach(resultLedgers::add);
+        return metadataCache
```

Comment:
we could return more data to the rest API, in order to show more info on the page that lists the ledgers